components/pedal-card-preview/index.js

```diff
@@ -14,25 +16,36 @@ import {
 } from "./pedal-card-preview.styled";
 
 const PedalCardPreview = ({ pedalData }) => {
+  const router = useRouter();
+
+  const handleImageClick = () => {
+    router.push(`/detail/${pedalData.id}`);
+  };
+
```

Comment:
You don't need that. The `Link` component handles the link for you. You can remove this complety and the `onClick` on the `Image` component as well.

Reply:
oh, nice to know, thanks! fixed